### PR TITLE
systemctl: add shutdown --show option

### PR DESCRIPTION
**Pull request by lnussel**
Shows the scheduled shutdown action and time if there's one.

### CODE REVIEW

**Comment by lnussel** [comment]
Not sure what's the most obvious choice for an option here. show (-s), query (-q) or list (-l)? -q is often used for quiet though.

**Comment by bluca** [comment]
Wouldn't a long option (`--show`) be more readable?

**Comment by medhefgo** [comment]
shutdown is a legacy binary we provide for compatibilty. It should not be getting any new features.

**Comment by bluca** [comment]
oh right, I missed that this was for the shutdown command rather than systemctl itself - then yes @medhefgo is right. Can this be added to systemctl reboot/poweroff/etc instead?

**Comment by medhefgo** [comment]
> oh right, I missed that this was for the shutdown command rather than systemctl itself - then yes @medhefgo is right. Can this be added to systemctl reboot/poweroff/etc instead?

It's adding it for both. So the shutdown part just needs to be removed. Also needs update to manpage.

**Comment by lnussel** [comment]
The long option probably is nicer indeed though

**Comment by keszybz** [comment]
I think it is OK to add this. `shutdown` is a compat command, but effectively we're the new owners of it on Linux, since most people are using our implementation. But please call the option `--show`. This reduces the risk of a conflict with another implementation.

(FWIW, I don't think `shutdown` is going away. Similarly to the `3` and `5` options on the kernel command line, things which are convenient will be always used.)

But please add the man page entry.

**Comment by lnussel** [comment]
Looks like  I went into a rabbit hole looking at the shutdown handling :/
Anyway, one thing at a time. Renamed to --show and added mention in man page.

**Comment by bluca** [comment]
Please update shell-completion/ too

**Comment by lnussel** [comment]
Looks like there is no existing shell completion for the shutdown command? Only systemctl is covered but the change doesn't impact systemctl in a user visible way.

File: man/shutdown.xml [hunk 1]
```diff
@@ -125,6 +125,13 @@
         <literal>now</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--show</option></term>
+
+        <listitem><para>Show a pending shutdown action and time if
+        there is any</para></listitem>
```
Comment (keszybz) [inline man/shutdown.xml]: Missing '.'.

**Review by keszybz — CHANGES_REQUESTED** [review]

File: man/shutdown.xml (hunk 1 above)
Comment (lnussel) [reply man/shutdown.xml]: eagle eyes :+1:

**Pull request merged**